crates/bitwarden-exporters/src/cxf/import.rs

```diff
@@ -126,6 +127,16 @@ pub(super) fn parse_item(value: Item) -> Vec<ImportingCipher> {
         add_item(CipherType::Identity(Box::new(identity)), custom_fields);
     });
 
+    // SSH Key credentials
+    if let Some(ssh) = grouped.ssh.first() {
+        match to_ssh(ssh) {
+            Ok((ssh_key, fields)) => add_item(CipherType::SshKey(Box::new(ssh_key)), fields),
+            Err(_) => {
+                // Include information about the failed items, or import as note?
+            }
```

Comment:
Yeah we need to figure out a strategy to handle errors in a more structured way

Reply:
We don't need to change the approach in this PR, but I do think we should opt for the "don't loose any data" approach and save it to a note, if we can encode the value in a meaningful way.